spec/fabricators/application_failure_fabricator.rb

```diff
@@ -0,0 +1,12 @@
+# Simulates Temporal::Connection::Serializer::Failure
+Fabricator(:application_failure, from: Temporal::Api::Failure::V1::Failure) do
```

Comment:
nit: Can you please move this one over to `fabricators/grpc` and prefix with `api_`? Helps with distinguishing between domain and edge entities